src/components/IdCard.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import './IdCard.css';
+
+class IdCard extends Component {
+    render() {
```

Comment:
Cuando vayas a escribir más de una vez `this.props.whatever`, piensa en destructurarlo.

`const { firstName, lastName, gender, height, birth } = this.props`

Te lo escribo en esta linea porque en este caso lo deberías hacer dentro del render pero fuera del return